src/scss/grommet-core/_objects.layer.scss

```diff
@@ -111,6 +111,14 @@
       max-height: calc(100vh - #{double($inuit-base-spacing-unit)});
       max-width: calc(100vw - #{double($inuit-base-spacing-unit)});
       transform: translate(50%, 50%);
+
```

Comment:
@alansouzati Ok, I've switched to deleting those lines. I can also confirm removing them fixes the issue in my application as well. Thank you!